+logging/logging.m

```diff
@@ -88,33 +88,55 @@ function setCommandWindowLevel(self, level)
     function setLogLevel(self, level)
       self.logLevel = level;
     end
+    
+    function tf = ignoreLogging(self)
+        tf = self.logLevel + self.commandWindowLevel == 2*self.OFF;
+    end

```

Comment:
Thanks. This feature is really necessary. I feel `getCallerInfo()` should perform the check though (and check instead that both levels are `OFF` instead of adding them up), and return immediately. The `ignoreLogging()` method seems redundant with disabling logging by setting the level to `OFF`.